src/Molecules/ToastMessage/ToastMessage.styled.ts

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+import { Box, Typography } from '../..';
+
+export const BoxContainer = styled(Box)`
+  background-color: ${({ theme }) => theme.colorTokens.neutral.background_focus};
+  padding: ${(p) => p.theme.spacing.unit(3)}px;
+  border-radius: ${(p) => p.theme.borderRadius8};
```

Comment:
use destructuring on all places here for consistency
```suggestion
  background-color: ${({ theme }) => theme.colorTokens.neutral.background_focus};
  padding: ${({ theme }) => theme.spacing.unit(3)}px;
  border-radius: ${({ theme }) => theme.borderRadius8};
```